RELEASE-NOTES.md

```diff
@@ -183,7 +187,6 @@ Changes in the `Entity` hierarchy:
 * Changed the constructor signature of `Property`
 * Removed `Entity::setClaims` (`Item::setClaims` has been retained)
 * Removed `Entity::stub`
-* Removed `Property::newEmpty`
```

Comment:
Did you click on the expand link on the left? I also was confused by this removal but it is part of the release notes of our Version 1.0 and github just makes it look somewhat strange.